deye_daemon.py

```diff
@@ -90,12 +91,29 @@ def __remove_duplicated_reg_ranges(self, reg_ranges: list[SensorRegisterRange])
         return result
 
 
+class setInterval :
```

Comment:
Please rename this class to something more meaningful, and start the classname with uppercased letter.

Reply:
Renamed the class to `IntervalRunner`.